### PR TITLE
Correct broken links, missing formatting, typos

### PR DESCRIPTION
**Pull request by carehart**

### CODE REVIEW

**Comment by carehart** [comment]
BTW, there is a last issue that could be dealt with, in the section near the bottom on filing issues for other than Dev Env's. I didn't want to presume to correct that, but someone should consider updating it (what to link to, whether to link to, whether to change the reference to "win-in"). And I suppose I could/should create a new issue for this, but I just figured I'd mention it here in case you may want to attend to it while considering the improvements I have proposed.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # Dev Environments
-This repository contains a feedback and issue tracker for Docker Dev Environments
+This repository contains a feedback and issue tracker for Docker Dev Environments.
 
-Getting Docker Environments
-Docker Dev Environments are free as part of Docker Desktop for Mac and Windows
-To download for Windows. To download for Mac.
+## Getting Docker Environments
+Docker Dev Environments are free as part of Docker Desktop for Mac and Windows ([download](https://www.docker.com/products/docker-desktop).
```
Comment (felipecruz91) [inline README.md:5]: Missing parenthesis at the end of the sentence.

```suggestion
Docker Dev Environments are free as part of Docker Desktop for Mac and Windows ([download](https://www.docker.com/products/docker-desktop)).
```

**Review by felipecruz91 — CHANGES_REQUESTED** [review]
Thank you @carehart for taking the time to correct the README. It looks good overall - there's just a missing parenthesis at the end of one sentence.

**Review by felipecruz91 — APPROVED** [review]

**Review by gtardif — APPROVED** [review]

**Comment by carehart** [comment]
Thanks for the correction and the merge. About the comment I'd offered, any thoughts?

**Comment by felipecruz91** [comment]
We've opened a [PR](https://github.com/docker/dev-environments/pull/95) including the GitHub repository links.

**Comment by carehart** [comment]
Ah great, thanks.

**Pull request merged**